evm/deploy-non-upgradable.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+require('dotenv').config();
+
+const {
+    Wallet,
+    getDefaultProvider,
+    utils: { isAddress },
+} = require('ethers');
+const readlineSync = require('readline-sync');
+const { getCreate3Address } = require('@axelar-network/axelar-gmp-sdk-solidity');
+const { Command, Option } = require('commander');
+const chalk = require('chalk');
+
+const { printInfo, writeJSON, isString, isNumber, isAddressArray, deployCreate3 } = require('./utils');
+
+async function getConstructorArgs(contractName, config) {
+    const contractConfig = config[contractName];
+
+    switch (contractName) {
+        case 'InterchainGovernance': {
+            const gateway = config.AxelarGateway?.address;
+
+            if (!isAddress(gateway)) {
+                throw new Error(`Missing AxelarGateway address in the chain info.`);
+            }
+
+            const governanceChain = contractConfig.governanceChain;
+
+            if (!isString(governanceChain)) {
+                throw new Error(`Missing InterchainGovernance.governanceChain in the chain info.`);
+            }
+
+            const governanceAddress = contractConfig.governanceAddress;
+
+            if (!isAddress(governanceAddress)) {
+                throw new Error(`Missing InterchainGovernance.governanceAddress in the chain info.`);
+            }
+
+            const minimumTimeDelay = contractConfig.minimumTimeDelay;
+
+            if (!isNumber(minimumTimeDelay)) {
+                throw new Error(`Missing InterchainGovernance.minimumTimeDelay in the chain info.`);
+            }
+
+            return [gateway, governanceChain, governanceAddress, minimumTimeDelay];
+        }
+
+        case 'AxelarMultisigMintLimiter': {
```

Comment:
switch to multisig

Reply:
I agree that's more concise but the contract name is pulled directly from the command line and is used to get the implementation file path. I can add an if-statement to look for AxelarMultisigMintLimiter.json if contract name is Multisig or leave as is, thoughts?

Reply:
there is no special mint limiter contract anymore though. We switched to [multisig](https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/contracts/governance/Multisig.sol) as a general purpose contract

Reply:
ok that makes sense, I'll change it then